src/loader/load.cc

```diff
@@ -22,12 +23,14 @@ std::vector<std::unique_ptr<loader_interface>> get_loaders() {
   return loaders;
 }
 
-timetable load(std::vector<std::pair<std::string, loader_config>> const& paths,
-               finalize_options const& finalize_opt,
-               interval<date::sys_days> const& date_range,
-               assistance_times* a,
-               shapes_storage* shapes,
-               bool ignore) {
+timetable load(
+    std::vector<std::tuple<std::string, std::string, loader_config>> const&
```

Comment:
```suggestion
    std::vector<std::tuple<std::string, std::string, loader_config>> const&
```

maybe a struct would make it more obvious and help to not swap the `std::string` parameters in the tuple by accident?